website/source/api/acl/legacy.html.md

```diff
@@ -289,4 +289,103 @@ $ curl \
+$ curl \
+    --request GET \
+    http://127.0.0.1:8500/v1/acl/replication
+```
+
+### Sample Response
+
+```json
+{
+  "Enabled": true,
+  "Running": true,
+  "SourceDatacenter": "dc1",
+  "ReplicationType" : "tokens",
+  "ReplicatedIndex": 1976,
+  "ReplicatedTokenIndex": 2018,
+  "LastSuccess": "2018-11-03T06:28:58Z",
+  "LastError": "2016-11-03T06:28:28Z"
+}
+```
+
+- `Enabled` - Reports whether ACL replication is enabled for the datacenter.
+
+- `Running` - Reports whether the ACL replication process is running. The process
+  may take approximately 60 seconds to begin running after a leader election
+  occurs.
+
+- `SourceDatacenter` - The authoritative ACL datacenter that ACLs are being
+  replicated from and will match the
+  [`primary_datacenter`](/docs/agent/options.html#primary_datacenter) configuration.
+
+- `ReplicationType` - The type of replication that is currently in use.
+
+   - `legacy` - ACL replication is in legacy mode and is replicating legacy ACL tokens.
+
+   - `policies` - ACL replication is only replicating policies as token replication
+     is disabled.
+
+   - `tokens` - ACL replication is replicating both policies and tokens.
+
+- `ReplicatedIndex` - The last index that was successfully replicated. Which data
+  the replicated index refers to depends on the replication type. For `legacy`
+  replication this can be compared with the value of the `X-Consul-Index` header
+  returned by the [`/v1/acl/list`](/api/acl/legacy.html#acl_list) endpoint to
+  determine if the replication process has gotten all available ACLs. When in either
+  `tokens` or `policies` mode, this index can be compared with the value of the
+  `X-Consul-Index` header returned by the [`/v1/acl/polcies`](/api/acl/policies.html#list-policies)
+  endpoint to determine if the policy replication process has gotten all available
+  ACL policies. Note that ACL replication is rate limited so the indexes may lag behind
+  the primary datacenter.
+
+- `ReplicatedTokenIndex` - The last token index that was successfully replicated.
+   This index can be compared with the value of the `X-Consul-Index` header returned
+   by the [`/v1/acl/tokens`](/api/acl/tokens.html#list) endpoint to determine
+   if the replication process has gotten all available ACL tokens. Note that ACL
+   replication is rate limited so the indexes may lag behind the primary
+   datacenter.
+
+- `LastSuccess` - The UTC time of the last successful sync operation. Since ACL
+  replication is done with a blocking query, this may not update for up to 5
+  minutes if there have been no ACL changes to replicate. A zero value of
+  "0001-01-01T00:00:00Z" will be present if no sync has been successful.
+
+- `LastError` - The UTC time of the last error encountered during a sync
+  operation. If this time is later than `LastSuccess`, you can assume the
+  replication process is not in a good state. A zero value of
+  "0001-01-01T00:00:00Z" will be present if no sync has resulted in an error.
```

Comment:
This isn't missing, it's just not under "Legacy" because it didn't change and still works the same for new ACLs. See https://www.consul.io/api/acl/acl.html#check-acl-replication.

Was there a specific concern that somone couldn't find it? We might consider breadcrumbs somewhere on this page if so but I think it's confusing to repeat the docs here as it's not "legacy" and still fully supported.

Reply:
@banks : The issue was that for folks that are still using the Legacy ACL HTTP  weren't able to find the ACL replication information, hence the idea of adding it to the Legacy section. Though I do see us adding breadcrumbs somewhere specifying that the same matter of checking ACL Replication works in both New & Legacy system in lieu of repeating the docs

Reply:
@kaitlincarter-hc @judithpatudith either of you have an opinion on what to do here?

Reply:
TLDR - We should revert this doc to the original before 1.4 was released so it's truly legacy information.

I think we should at least add breadcrumbs. My general concern about the legacy documentation is that the goal is unclear. Since the old system will not change, this page should theoretically never need updates (kind of like versioned docs). So, we shouldn't need to be concerned about duplicating info that will need to be updated in multiple spots. I did notice we changed some parameter names to the updated versions. I am glad we changed them back since users on 1.3 and older will still need reference documentation. 

I can also double check that the warnings at the top of this page are clear that users on 1.4 and newer should use the other docs.